test/src/test_vaildate_generated_sq.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+import os
+import json
+import jsonschema
+
+schema_file = "json-schema/StructuredQuery-schema-v3.json"
+generated_sqs_dir = "test/resources/generated-sqs/"
+
+@pytest.fixture
+def schema():
+    loaded_schema = None
+    with open(schema_file, 'r') as file:
+        loaded_schema = json.load(file)
+    return loaded_schema
+
+def test_validate_sqs(schema):
+    if not os.path.exists(generated_sqs_dir):
```

Comment:
Does the test fail if the SQ dir doesn't exist?

Reply:
should it fail if the directory doesn't exist?

Reply:
I would say yes, because we like to validate at least one query.